pallets/pool-system/src/tests/mod.rs

```diff
@@ -20,15 +20,10 @@ use cfg_types::{
 use frame_support::{assert_err, assert_noop, assert_ok};
 use orml_traits::asset_registry::{AssetMetadata, Inspect};
 use rand::Rng;
-use sp_core::{storage::StateVersion, Encode};
+use sp_core::storage::StateVersion;
 use sp_runtime::{
-	traits::{ConstU32, One, Zero},
-	FixedPointNumber, Perquintill, TokenError, WeakBoundedVec,
-};
-use xcm::{
```

Comment:
It looks like `xcm` has been removed from both places it's used in the `pool-system` pallet.  I think we can remove it from the pallets `Cargo.toml`.

Reply:
Done in https://github.com/centrifuge/centrifuge-chain/pull/1340/commits/0bf232376e5723ed78f3eed59297f20def740a8f ✅